presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -147,6 +148,10 @@ public void testArrayToJson()
         assertFunction("cast(cast(ARRAY[123456789, -123456789, null] AS ARRAY<INTEGER>) AS JSON)", JSON, "[123456789,-123456789,null]");
         assertFunction("cast(cast(ARRAY[1234567890123456789, -1234567890123456789, null] AS ARRAY<BIGINT>) AS JSON)", JSON, "[1234567890123456789,-1234567890123456789,null]");
 
+        assertFunction("cast(cast(ARRAY[1, 2, null] AS ARRAY<REAL>) AS JSON)", JSON, "[1.0,2.0,null]");
```

Comment:
These tests are not necessary in this PR. `ARRAY<REAL> TO JSON` is already supported.